Descent3/WeaponFire.cpp

```diff
@@ -1183,7 +1183,7 @@ void AquireElectricalTarget(object *obj) {
     fq.startroom = obj->roomnum;
     fq.p1 = &dest;
     fq.rad = .0001f;
-    fq.thisobjnum = Objects - obj;
+    fq.thisobjnum = OBJNUM(obj); // bugfix?
```

Comment:
unclear what this comment means - this doesn't look markedly different from the other `OBJNUM` pattern usages. what's the bug?

Reply:
ah i see the reversed `Objects - obj` instead of `obj - Objects`